src/V3File.h

```diff
@@ -165,6 +165,9 @@ class V3OutFormatter VL_NOT_FINAL {
     void blockDec() {
         if (!m_parenVec.empty()) m_parenVec.pop();
     }
+    void ensureNewLine() {
+        if (!m_nobreak) puts("\n");
+    }
```

Comment:
Likewise you can please merge this into master (along with DPI spacing).

Reply:
I think this is only necessary becasuse #define needs to go on a new line, so if we are not using them, we can loose this, though might be handy to make the output prettier. In any case, this is not used outside of this patch, so probably should not be separate (as it would then be unused). Note fixing the trailing whitespace was necessary to make that condition work I will merge that separately as requested.